app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+    def index
+        @foods = FoodFacade.search_foods(params[:q])
+        @results = FoodFacade.total_results(params[:q]) 
+    end
```

Comment:
both of these facade methods call the same service, which feels a bit unnecessary. I think you could refactor this to just call one facade method that will in turn return both the foods, and the total results. 

You could look into using the following syntax:

@results, @foods= FoodFacade.your_method(params[:q])

You could put this in your controller, and then have your_method return a 2d array where the first element was the totalHits, and the second element was the collection of food objects you created.

The call to get all the food data comes back with 'totalhits', so no need to call the facade again, which in then would call the service again.